index.js

```diff
@@ -224,6 +236,38 @@ const fillFieldWithTextWhenVisibleAndPressEnter = function(
   );
 };
 
+const scrollToElement = function(htmlElement)
+{
+  browser.executeScript('arguments[0].scrollIntoView(true);', htmlElement);
+};
+
+const scrollToElementWhenVisible = function(
+  htmlElement,
+  message = getDefaultIsNotVisibleMessage(htmlElement),
+  timeout = DEFAULT_TIMEOUT_IN_MS
+) {
+  this.waitForElementVisibility(htmlElement, message, timeout);
+  this.scrollToElement(htmlElement);
+};
+
+const scrollToElementWhenPresent = function(
```

Comment:
Why do you need to scroll to an element when present? There are cases when an element can be present but not visible. Don't you think it would be better to leave only the `scrollToElementWhenVisible` function?